sdk/test/opentelemetry/sdk/trace/export/simple_sampled_span_processor_test.rb

```diff
@@ -7,4 +7,95 @@
 require 'test_helper'
 
 describe OpenTelemetry::SDK::Trace::Export::SimpleSampledSpanProcessor do
+  export = OpenTelemetry::SDK::Trace::Export
+
+  let :mock_span_exporter do
+    mock = Minitest::Mock.new
+    def mock.nil?
+      false
+    end
+
+    mock
+  end
+
+  let :stub_span_unsampled do
```

Comment:
Is there a reason why an `OpenStruct` is better than a concrete `Span` and `SpanContext`?

Reply:
Ah -- the reason seems to be me not paying quite enough attention!

I saw the TODO comment and empty class body in https://github.com/open-telemetry/opentelemetry-ruby/blob/master/sdk/lib/opentelemetry/sdk/trace/span.rb#L11, and so I thought this was not yet implemented and stubbed out just enough to make the tests pass.

Looking again, I missed how that inherits from the pieces in /api/, which do have implementations. I'll swap this out for concrete Span and SpanContexts tomorrow.

Reply:
I've updated this in the new version of the PR.